packages/hs-react-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -198,6 +208,12 @@ const Dropdown = ({
 }: DropdownProps): JSX.Element | null => {
   const [isOpen, setIsOpen] = useState<boolean>(false);
 
+  // Merge the default styled continer prop and the placeholderProps object to get user styles
+  const placeholderMergedProps = {
+    ...{ StyledContainer: PlaceholderContainer },
+    ...placeholderProps,
+  };
```

Comment:
Should this be the StyledValueContainer we're spreading?

Reply:
No, this is for the placeholder specifically. The placeholder uses the Text component and this is to apply the default styling and override that by any of the placeholderProps that get passed in. I figured for most use cases they wouldn't want to customize the placeholder, but it's easy enough to do with the placeholderProps prop.